packages/flutter_tools/test/general.shard/protocol_discovery_test.dart

```diff
@@ -24,7 +24,7 @@ void main() {
         ipv6: false,
         hostPort: null,
         devicePort: null,
-        throttleDuration: const Duration(milliseconds: 200),
+        throttleDuration: const Duration(milliseconds: 5),
```

Comment:
I didn't look at what it would take to remove this timeout/delay, but I did make it much smaller

Reply:
Oh, weird.  Does `Duration.zero` work?

Reply:
if its zero the throttling logic doesn't quite work